atomsci/ddm/pipeline/feature_importance.py

```diff
@@ -203,7 +194,7 @@ def base_feature_importance(model_pipeline=None, params=None):
 
     # Get the training, validation and test sets (we assume we're not using K-fold CV). These are DeepChem Dataset objects.
     (train_dset, valid_dset) = model_data.train_valid_dsets[0]
-    test_dset = model_data.test_dset
+    _test_dset = model_data.test_dset
```

Comment:
this function only deals with the training dataset; documentation should be updated

Reply:
I updated the documentation, but this needs a test. https://github.com/ATOMScience-org/AMPL/commit/47ac68e569cfd6f7b5513a0713303bd6e658ff16